src/caballo.js

```diff
@@ -0,0 +1,18 @@
+const nivel = require("./nivel")
+
+class caballo{
+    /**
+     * Constructor clase caballo
+     * @param {String} nombre Identificador del caballo 
+     * @param {nivel} nivel Nivel del caballo
+     * @param {Boolean} descansado Caballo está descansado o no
+     */
+    constructor(nombre, nivel, descansado){
+        this.nombre = nombre;
+        this.nivel = nivel;
+        this.descansado = descansado;
```

Comment:
Mientras que los ficheros tienen que estar en mayúsculas, la buena práctica en JavaScript es que las clases estén en mayúsculas. Una vez más, consultar *todas* las buenas prácticas y no la última que os haya dicho que cambiéis.
Por otro lado, ¿esto es un objeto valor o una entidad? ¿Qué diferencia hay entre ellas?